eo-runtime/src/main/java/EOorg/EOeolang/EOcage.java

```diff
@@ -61,7 +62,7 @@ public EOcage(final Phi sigma) {
      * @since 0.17
      */
     @XmirObject(oname = "cage.write")
-    private final class Write extends PhDefault {
+    private static final class Write extends PhDefault {
```

Comment:
@maxonfjvipon Why do you make it static?

Reply:
@Graur IDEA suggested, I fell for it :)